stellargraph/connector/neo4j/graph.py

```diff
@@ -0,0 +1,97 @@
+__all__ = ["Neo4jStellarGraph"]
+
+import numpy as np
+import scipy.sparse as sps
+from ...core.experimental import experimental
+
+
+@experimental(reason="the class is not fully tested and lacks documentation")
+class Neo4jStellarGraph:
```

Comment:
as discussed, this could derive from a class with some of the StellarGraph functionality?  (potentially follow-on work. in another PR - that would require a new class, something like "AbstractStellarGraph", which StellarGraph would also need to inherit from.

Reply:
yeah i think there's defeinitly some shared functionality like creating the adjacency matrix